helping-and-teaching.md

```diff
@@ -1,25 +1,21 @@
-# Tips for helpers
+# Tips for exercise leaders
 
-This page contains the most important general hints for helpers for
-helpers.  See the {doc}`helper-intro` for more.
+This page contains the most important general tips for exercise leaders before during and after the workshop. If you have not visited the main introduction page, follow [this link](helper-intro).
```

Comment:
The link in the preview goes to https://github.com/coderefinery/manuals/blob/master/helper-intro which ends up with 404 error. Maybe you need to follow the grammar {doc}`helper-intro` here?
```suggestion
This page contains the most important general tips for exercise leaders before during and after the workshop. If you have not visited the main introduction page, follow {doc}`helper-intro`.
```

Reply:
@rkdarst , maybe I am wrong. Can you help here?